mac.go

```diff
@@ -45,13 +46,11 @@ func verifyMac(macData *macData, message, password []byte) error {
 	expectedMAC := mac.Sum(nil)
 
 	if !hmac.Equal(macData.Mac.Digest, expectedMAC) {
-		return PasswordIncorrect("Incorrect password, MAC mismatch")
+		return IncorrectPasswordError
 	}
 	return nil
 }
 
-// PasswordIncorrect Error indicates that the supplied password is incorrect.
+// IncorrectPasswordError is returned when an incorrect password is detected.
 // Usually, P12/PFX data is signed to be able to verify the password.
-type PasswordIncorrect string
-
-func (e PasswordIncorrect) Error() string { return string(e) }
+var IncorrectPasswordError = errors.New("pkcs12: decryption password incorrect")
```

Comment:
Yah. This could be the only spot in the standard library that has it wrong (based on a search for `var [A-Z]\w+Error`).

So going the *correct* way vs. going the way that matches DecryptPEMBlock?